### DISCUSSION TITLE
Support for the New Teams App?

### DISCUSSION THREAD

**Discussion by joshvermette** [tom-englert/RegionToShare#70] [Q&A]
I have recently switched to an ultrawide + portrait monitor setup so just recently started (trying) to use RTS full time.  I am having some unexpected behavior where users don't see new windows that I drag into the frame.  I am also using the new teams app and wondering if it is fully supported? I couln't find any documentation or issues specifically related to the new version.

**Comment by tom-englert** [comment]
> I am having some unexpected behavior 

That's too vague to give you some comprehensive answer.

Considering https://github.com/tom-englert/RegionToShare#how-it-works, there is nothing special RTS can do for whatever you mean by "new teams app".

**Reply by joshvermette** [reply]
Hi Tom, 

Happy to add more context.  MS GA'd what they are calling 'new teams' (vs classic teams) in Oct 2023. This is a completely different version of the original teams app. I didn't see anything in the releases or change logs that mentioned it so wasn't sure if it had full support as I have been experiencing some issues when using it.    I will do some testing on both versions and see if I can identify if the new app is the root cause or if it is something else entirely

Landing page: https://adoption.microsoft.com/en-us/new-microsoft-teams/

GA release announcement: https://techcommunity.microsoft.com/t5/microsoft-teams-blog/announcing-general-availability-of-the-new-microsoft-teams-app/ba-p/3934603

**Comment by alxire** [comment]
I am in a similar situation. I also just switched to an ultrawide (LG 40WP95C-W) and I'm on the new Teams as well. I have experienced the same issue as you but the biggest issue is the mouse lag. I do a lot of demoing via Teams and I cannot use RTS to do this bc the mouse lag is so bad. I really wanted to like this bc the concept is fantastic but not supporting my primary use case (well) is a bummer. I am forced to have a secondary monitor which I use to share or just share a single window which sucks. Hopefully its me or something I am doing but I doubt that is the case.

**Reply by joshvermette** [reply]
I went back to the old teams in the interim and its been working great.  Hopefully he can get it updated to work better with the new teams as it rolls out to more people.

**Reply by tom-englert** [reply]
> the biggest issue is the mouse lag

By default the mouse is not part of the RTS output, but is directly captured by teams - so I'm not sure what RTS could do about this?

**Comment by alxire** [comment]
Yeah.  Would be nice. Unfortunately I don't think I can even rollback. Did
you experience any mouse lag when sharing?

**Reply by joshvermette** [reply]
Luckily I'm one of the ones in my org that gets access to monthly releases so I was able to opt in/out of it. I don't remember mouse lag but I went back pretty quick. If I get some time I will retest it and let you know

**Comment by alxire** [comment]
Huh, well thanks. I can actually rollback via a little toggle option. Your
comment made me look a little closer. Once I rolled back I started a
meeting and shared, still lagging but not as bad. But bad enough that I
won't use it. Maybe its the Logi software I installed recently to use the
new mouse I got. Unfortunately I installed Logi+ the same day I got Fancy
Zones and RTS. Process of elimination time I suppose.

**Reply by joshvermette** [reply]
Check the CPU utilization when its running, that may be the cause. I have a pretty beefy laptop and it still uses 15-20% of my processor. I am also running PowerToys and Logi Options. best of luck

**Comment by alxire** [comment]
I have actually, it will peak at 15%, and same- beefy laptop
(surface studio LT w. dedicated nvidia gpu). I feel like it should be
fine...How many monitors total are you running?

**Reply by joshvermette** [reply]
Surface book 3 (i7, fully loaded) running 2 monitors, 1 1440p (2560 x 1440) and one UW (3440 x 1440)

**Comment by alxire** [comment]
I'm not sure either nor am I sure that it is RTS causing the issue. It's
just an observation that while using teams to share, logi for mouse and RTS
with an ultra wide causes a mouse delay that makes it difficult to use if
I'm doing things like a demo where it's mouse intensive.
